### DISCUSSION TITLE
Why does documentation for AWS EC2 specify Ubuntu?

### DISCUSSION THREAD

**Discussion by GerardKetuma** [gitpod-io/openvscode-server#133] [General] [answered]
Does this only work for Ubuntu if we are running it in the cloud? Or will this work with any linux distro?

**Comment by filiptronicek** [comment, marked as answer]
Ubuntu is there because most people know Ubuntu, it's easy to use and there is a lot of help online. 

The process that powers OpenVSCode server is just a Node process, so if you can run Node and bash on your distro, it should work simply fine. If you encounter any issues, feel free to reach out!

**Reply by berk-efe** [reply]
hello i cant find documentation for AWS EC2 can you help please?

**Reply by filiptronicek** [reply]
Hey there @AkioKiyota, you can find the guide under the `docs` branch: https://github.com/gitpod-io/openvscode-server/tree/docs/guides/aws-ec2.
